src/components/Text/index.ts

```diff
@@ -0,0 +1,4 @@
+import {Text as TextComponent} from './Text';
+import {createHOC} from 'wix-ui-core/dist/src/createHOC';
```

Comment:
Why are u doing this ?

Reply:
In wix-stylr-react people pass `dataHook` today, so if we won't wrap it with the HOC they will break

Reply:
sucks, I would prefer not doing that, maybe we should have a task to remove it with the next major version ?

Reply:
It will be an annoying breaking change for all the users, that is for sure. We can discuss it in general when to do that, we better off without it.
But since the Badge & CounterBadge are in wix-style-react already, they alos use this HOC.

Until we decide what is the plan to deprecate it, we must continue to wrap it